core/src/backend/navigator.rs

```diff
@@ -682,3 +690,14 @@ pub fn fetch_path<NavigatorType: NavigatorBackend>(
         Ok(response)
     })
 }
+
+/// Parses and returns the encoding out of an HTTP header content type string
+/// if existing.
+pub fn get_encoding(content_type: String) -> Option<&'static Encoding> {
```

Comment:
should be possible to pass `&str` without converting `.to_string()` here, I think

Reply:
Yeah, that's true! Thank you :)